App/Logic/DataUploadLogic.swift

```diff
@@ -172,12 +172,12 @@ extension Logic.DataUpload {
         } catch NetworkManager.Error.unauthorizedOTP {
           // User is not authorized. Bubble up the error to the calling ViewController
           try await(context.dispatch(Logic.Loading.Hide()))
-            try context.awaitDispatch(ShowCunErrorAlert(message: L10n.UploadData.ErrorCun.message))
+            try context.awaitDispatch(ShowCunErrorAlert(title: L10n.UploadData.VpnError.title, message: L10n.UploadData.ErrorCun.message))
           throw Error.verificationFailed
         } catch NetworkManager.Error.otpAlreadyAuthorized {
             // cun Already Authorized. Bubble up the error to the calling ViewController
             try await(context.dispatch(Logic.Loading.Hide()))
-            try context.awaitDispatch(ShowCunErrorAlert(message: L10n.UploadData.UnauthorizedCun.message))
+            try context.awaitDispatch(ShowCunErrorAlert(title: L10n.UploadData.UnauthorizedCun.title, message: L10n.UploadData.UnauthorizedCun.message))
```

Comment:
<!--
  0 failure: 
  1 warning:  swiftlint (0.39.2...
  
  
  DangerID: danger-id-Danger;
  File: App/Logic/DataUploadLogic.swift;
  Line: 180;
-->

- :warning: swiftlint (0.39.2): Line should be 130 characters or less: currently 152 characters (line_length)